vdb/__init__.py

```diff
@@ -492,7 +492,8 @@ def notify(self, event, trace):
             trace.setMeta('PendingBreak', False)
             bp = trace.getCurrentBreakpoint()
             if bp:
-                self.vprint("Thread: %d Hit Break: %s" % (tid, repr(bp)))
+                if not bp.silent:
```

Comment:
adding the ability to make "silent" bps which don't alert when hit

Reply:
i'll do ya one better..  i'll make it so all breakpoints can be listed using `bp -A` and i'll make `bpedit` allow the setting of fastbreak and silent (and add a verbose list).

Reply:
but this particular code isn't to hide the existence of the breakpoint, but to keep from spitting out the alert that we hit it.

Reply:
done